### DISCUSSION TITLE
Duda GGpairs

### DISCUSSION THREAD

**Discussion by Asier-Camara** [DatSciR/ciencia_datos#7] [Q&A] [answered]
Hola Julen y Vero,

Quería consultaros una duda sobre el gráfico ggpairs, tengo hecho este con mis datos, pero a la hora de describirlo no encuentro que valor de significancia o que significan exactamente los asteriscos (que estadística hace) y la cantidad que aparecen, así como los puntos pequeños a lado del número. 

Os agradecería si podéis echarme una mano,

Asier
![Duda_ggpairs](https://github.com/user-attachments/assets/78dc2321-2d4f-4d26-bedd-1ff5622e1dc8)

**Comment by VeruGHub** [comment, marked as answer]
Hola Asier:

Hace poco estuve buscando lo mismo. La verdad es que no lo tienen muy bien explicado.
ggpairs llama a a cor() para calcular las correlaciones (https://briatte.github.io/ggcorr/#correlation-methods) y a ggally_cor() para representarlas (https://ggobi.github.io/ggally/reference/ggally_cor.html). 
En este último enlace tienes lo que significan los astericos :)

Saludos
Vero

**Reply by Asier-Camara** [reply]
Hola Vero,

Genial! Me viene de lujo, muchas gracias

Saludos,
Asier

**Reply by Asier-Camara** [reply]
Hola de nuevo,

Quería hacer un gráfico de barras con el error estándar de los míticos 
![image](https://github.com/user-attachments/assets/6a2bae96-cf92-4e13-a899-fc17d20f6762).  En la chuleta no veo ninguno o no sé si está pero no lo identifico, y en internet he mirado pero me pierdo un poco. 

Alguien ha tenido que hacer o podría echarme una mano con algunos comandos fáciles?

Gracias

**Comment by Julenasti** [comment]
Hola Asier,
[Aquí](https://r-graph-gallery.com/4-barplot-with-error-bar.html) puedes ver un ejemplo de cómo hacerlo. De todas formas, los gráficos de barras tienen varios problemas y se desaconsejan cada vez más. Por ejemplo con lo que quieres hacer solo le das al lector información de la media +- desviación estándar/error estándar/intervalo de confianza pero nada sobre su distribución. Mejor hacer un boxplot, o mejor aún un violin plot, o mejor aún una combinación de ambos. Mi sugerencia sería que le echaras un vistazo al paquete [ggdist](https://mjskay.github.io/ggdist/articles/slabinterval.html)
Saludos

**Reply by Asier-Camara** [reply]
Sí, creo que me voy a decantar mejor por un boxplot, porque me resulta más fácil tb a nivel metodológico el hacerlos, además de que sí que aporta más información. 

Pero gracias por la info Julen, está bien saberlo por si alguna vez lo necesito. 

Eskerrik asko!

Asier
